phpunit/navigation-block-fallback-test.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Tests Fallback Behavior for Navigation block
+ *
+ * @package WordPress
+ */
+
+class Tests_Block_Navigation_Fallbacks extends WP_UnitTestCase {
+
+	public function set_up() {
+		parent::set_up();
+		// Navigation menu will be created on Theme switch. Therefore in order to test
+		// the behaviour of `gutenberg_block_core_navigation_create_fallback`
+		// the auto-creation of a fallback must be disabled for this initial
+		// theme switch.
+		add_filter( 'block_core_navigation_skip_fallback', '__return_true' );
+		switch_theme( 'emptytheme' );
+		remove_filter( 'block_core_navigation_skip_fallback', '__return_true' );
+	}
+
+	private function get_navigations_in_database() {
+		$navs_in_db = new WP_Query(
+			array(
+				'post_type'      => 'wp_navigation',
+				'post_status'    => 'publish',
+				'posts_per_page' => -1,
+				'orderby'        => 'date',
+				'order'          => 'DESC',
+			)
+		);
+
+		return $navs_in_db->posts ? $navs_in_db->posts : array();
+	}
+
+	private function mock_wp_install() {
+		$user = get_current_user();
+		do_action( 'wp_install', $user );
+	}
+
+	public function test_should_auto_create_navigation_menu_on_wp_install() {
```

Comment:
It is recommended to add `@covers` tags to the test methods, as this helps to correctly calculate the code coverage.
https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#annotations
Also: https://docs.phpunit.de/en/9.5/annotations.html#covers

```suggestion
	/**
	 * @covers ::block_core_navigation_get_default_fallback
	 */
	public function test_should_auto_create_navigation_menu_on_wp_install() {
```